lerna.json

```diff
@@ -1,7 +1,13 @@
 {
-  "lerna": "2.0.0-beta.36",
   "version": "2.3.1",
   "command": {
+    "version": {
+      "allowBranch": [
+        "master",
+        "release/*"
```

Comment:
why not just use develop here instead of having to create a new release branch?

Reply:
Removed the need to create `release/*` branch. Updated this config accordingly.